nix/scripts/gcroots.sh

```diff
@@ -15,8 +15,13 @@ if [[ -z "${TARGET}" ]]; then
     exit 1
 fi
 
+if [[ -n "${NIX_SYSTEM}" ]]; then
+    systemArg="--argstr system ${SYSTEM}"
```

Comment:
I don't think adding yet another setting for developers to be aware of is a good idea. We should just set a `config.status-im.ci-build` variable when `CI=true` in the environment and set it implicitly in `build.sh` and `shell.sh` wrappers.